rapids-cmake/cpm/detail/package_details.cmake

```diff
@@ -92,7 +92,6 @@ function(rapids_cpm_package_details package_name version_var url_var tag_var sha
   set(exclude_from_all OFF)
   rapids_cpm_json_get_value(exclude_from_all)
 
-  set(always_download OFF)
```

Comment:
We should unset the variable here instead. That way any user variable called `always_download` won't trigger this incorrect behavior as well.

Reply:
Done.